angular/projects/admin-nrpti/src/app/records/orders/order-add-edit/order-add-edit.component.ts

```diff
@@ -47,7 +47,7 @@ export class OrderAddEditComponent implements OnInit, OnDestroy {
     private factoryService: FactoryService,
     private utils: Utils,
     private _changeDetectionRef: ChangeDetectorRef
-  ) {}
+  ) { }
```

Comment:
Something weird going on with your formatting settings.  It shouldn't be adding spaces between brackets like this.